core/chainlink.Dockerfile

```diff
@@ -73,10 +73,19 @@ FROM ubuntu:18.04
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y ca-certificates
 
-WORKDIR /root
+# Creating chainlink user
+RUN mkdir -p /home/chainlink/bin/
+RUN echo "chainlink:x:1000:" >> /etc/group
+RUN echo "chainlink::1000:1000:chainlink user:/home/chainlink:/bin/bash" >> /etc/passwd
```

Comment:
Can we use the `useradd` command here? That feels more expressive than manually inserting into the file. It should also handle both inserts.